js/app.js

```diff
@@ -0,0 +1,44 @@
+ const a = 4;
+
+ const b = 53;
+ 
+  const c = 57;
+  
+  const d = 16;
+
+  const e = 'Kevin';
+
+  a < b;
+c > d;
+'Name' _=_ 'Name';
+a + b = c;
+a + a >d;
```

Comment:
same as above -- 
try console.log'ging what you wrote for each of these in the developer tools console in google